build/config/compiler/BUILD.gn

```diff
@@ -533,13 +529,13 @@ config("runtime_library") {
       "-D__ANDROID_API__=$android_api_level",
     ]
 
-    lib_dirs += [ "$android_libcpp_root/libs/$android_app_abi" ]
-
-    libs += [
-      "$android_libcpp_library",
-      "c++abi",
+    include_dirs = [
+      "//third_party/libcxx/include",
```

Comment:
Also, having these in the public configs should make this include point moot.

Reply:
The compiler still needs these to resolve libcxx symbols in source_set targets. Those targets currently do not get an implicitly added dependency to libcxx (same thing in chrome: https://cs.chromium.org/chromium/src/build/config/posix/BUILD.gn?type=cs&g=0&l=47).